cmd/lakectl/cmd/annotate.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const annotateTemplate = `{{ range $val := .Commits}}
+{{ $.Object|ljust 10}} {{ $val.Committer }} {{ $val.Id | printf "%.16s"|ljust 20 }} {{ $val.Message |ljust 30 }} {{ $val.CreationDate|date }}{{ end -}}
+`
+
+var annotateCmd = &cobra.Command{
+	Use:   "annotate <path uri>",
+	Short: "Show who created the latest commit",
+	Long:  "Show who created the latest commit to an object or objects in a give path",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, c_args []string) {
```

Comment:
c_args - non Go naming - suggest to call it `args`

Reply:
Fixed as well.